.github/workflows/codeql.yml

```
@@ -0,0 +1,60 @@
+name: "CodeQL"
```

Name should be the exact same as the filename. This is a pattern that we use across our GitHub Actions to name workflows if we can keep it that way, thanks:

```suggestion
name: codeql
```